### ISSUE TITLE
Update framework with newest official update

### ISSUE THREAD

**Issue by JABirchall** [fkubis/teamspeak-php-framework#14]
planetteamspeak started to actively update their framework now mite be good idea to add what they have added to this version. https://github.com/planetteamspeak/ts3phpframework

**Comment by fkubis** [comment]
Yeah I die notice this. They moved to github. Iam currently thinking about closing this repo and improve the original by PR.

**Comment by JABirchall** [comment]
That would be nice as long as you made it PSR-4 compliant, else  I would extreme be unhappy about that.

**Comment by fkubis** [comment]
The offical Version 2.0 will use namespaces. Its would take to much time for me to figure out what the changes they did or refaktor the project again. So I will not update soon, maybe later.

**Comment by defaye** [comment]
So what is the verdict on this project? Should we just use planetteamspeak's one instead?

**Comment by JABirchall** [comment]
@defaye I prefer to use this project because of the PSR-4 when planetteamspeak release the new framework with the newest standards ill check it out. May even contribute to them.

**Comment by fkubis** [comment]
Take what you prefere.

Its not 100% clear what they did in the update. So the only way to get the changes to this project is do the covert to PSR-4 again, with is not realy easy and done in a hour. So I will not do this soon, maybe never. 

As they will support namespaces in version 2.0, I will close this project as they released it.

**Comment by JABirchall** [comment]
@fkubis I dunno, I think they have decided to make 2.0 in python. And I dont know python :/ So this project would have to live on.
